.github/actions/build/action.yaml

```diff
@@ -38,13 +47,23 @@ runs:
           cd src
           gn args --list --short --overrides-only out/${{ matrix.platform }}_${{ matrix.config }}
         shell: bash
+      - name: Ninja build test targets
+        if: inputs.upload_on_host_test_artifacts == 'true' || inputs.upload_on_device_test_artifacts == 'true'
+        env:
+          # TODO(bug): Replace hardcoded list with dynamically generated one.
+          TEST_TARGETS_JSON_FILE: .github/config/${{ matrix.platform }}.json
+        run: |
+          set -x
+          cd src
+          time autoninja -C out/${{ matrix.platform }}_${{ matrix.config }} $(cat "${TEST_TARGETS_JSON_FILE}" | jq -cr '.test_targets | join(" ")')
```

Comment:
Would be nice if there was just a `ninja -C blah cobalt_test_targets` in the future, but that's maybe too self-referential ( e.g. can't calculate gn deps chart while you are invoking gn, and can't have a ninja target calculate its own deps either ).

Reply:
We could set up such a fixed target in gn with the list of targets generated by the script. It could run in its own action where it would compare the list it calculates and fail if there's a mismatch.